portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,18 +37,20 @@
 @WebServlet("/data")
 public final class DataServlet extends HttpServlet {
 
-  private ArrayList<String> comments = new ArrayList<>(); 
   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private UserService userService = UserServiceFactory.getUserService();
+  private int maxComments = 10;
```

Comment:
Looks like this was removed in #8, why was this added back in?

Reply:
I realized I didn't need it then, but ended up needing it in this week's content.

Reply:
The same comment applies still, requests that specify a max-comments query parameter will affect subsequent requests without the query parameter specified